src/main/java/com/kustlik/medicalclinic/model/entity/Visit.java

```diff
@@ -0,0 +1,43 @@
+package com.kustlik.medicalclinic.model.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.time.LocalDateTime;
+
+@Builder
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString
+public class Visit {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(nullable = false)
+    private Long id;
+    LocalDateTime appointment;
+    int duration;
+    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
Jak jest to One to raczej nie wplynei na wydajnosc lazy czy eager bo i tak pobiera tylko 1 zaleznosc